tools/amd/build.js

```diff
@@ -37,7 +38,7 @@ export default function BuildBower() {
     .then(() => Promise.all([
       bowerConfig(),
       generateFactories(factoriesDestination, babelOptions),
-      exec(`babel ${babelOptions} ${srcRoot} --out-dir ${libDestination}`),
+      buildFolder(srcRoot, libDestination, babelOptions),
```

Comment:
What is the thought behind not using babel's cli? I don't understand why the more complicated approach is necessary. Does the babel-cli not work on windows?